cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -162,6 +162,12 @@
   expect(@content_type).to match(content_type)
 end
 
+Then(/^the HTTP response is base64 encoded$/) do
+  # Override encoded response with decode one to use other helpers
+  @result = @response_body = Base64.strict_decode64(@result)
+  expect(JSON.parse(@result).is_a?(Hash)).to be true
+end
```

Comment:
1. Some steps use `@result` as request_steps.rb,  and some use `@response_body` as authenticator_helpers.rb. Same happens with `@status` and `@http_status`. In order to prevent code reuse (of steps), I needed to use both `@result` and `@respone_body`. About the naming - valid base64 should be decode successfully. I need to override the response with decoded response in order to use other steps as the comments suggests.

2. A valid decoded token should be parsed by JSON. So if it fails during parsing, the test does its job. Do you prefer custom exception instead?